lib/SendGrid.php

```diff
@@ -13,6 +13,8 @@
  * @link      http://packagist.org/packages/sendgrid/sendgrid
  */
 
+namespace SendGrid;
```

Comment:
This was the only questionable change. PSR2 requires a namespace to be set, so I'm just using `/SendGrid` based on the `$namespace` variable set at the beginning of the class.